SECURITY.md

```diff
@@ -0,0 +1,16 @@
+# Security Policy
+
+## Supported Versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 6.x     | :white_check_mark: |
+| 5.x     | :x:                |
+| 4.x     | :x:                |
+| 3.x     | :white_check_mark: |
```

Comment:
We can drop 3.x.

Reply:
So all versions below the current version are dropped?

Reply:
Yes.